### PR TITLE
chore: update golang version(v1.16 -> v1.20) & exchange log to fmt and panic

### PR DESCRIPTION
**Pull request by Reimei1213**
- Update golang version (v1.16 -> v1.20)
- exchange log to fmt and panic

### CODE REVIEW

**Review by 0daryo — DISMISSED** [review]
👍

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,17 @@
 module github.com/abema/github-actions-merger
 
-go 1.16
+go 1.20
```
Comment (0daryo) [inline go.mod:3]: golang version is still 1.16.
Dockerfile
https://github.com/abema/github-actions-merger/blob/main/Dockerfile?L1#L1

Comment (Reimei1213) [reply go.mod:3]: 修正致しましたので、ご確認をよろしくお願いいたします:pray:
https://github.com/abema/github-actions-merger/pull/34/commits/e27f57f0a6201eea33673668fe704dc302f95075#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R1

**Review by 0daryo — APPROVED** [review]
👍

**Pull request merged**